### PR TITLE
Add support for deploying services from service definitions to the mi…

### PR DESCRIPTION
**Pull request by wwwtyro**
…gration process.

### CODE REVIEW

File: facade/service.go [hunk 1]
```diff
@@ -213,15 +213,19 @@ func (f *Facade) MigrateServices(ctx datastore.Context, request dao.ServiceMigra
...
+	if err = f.validateServiceDeploymentRequests(ctx, request.Deploy); err != nil {
 		return err
 	}
 
 	// If this isn't a dry run, make the changes.
 	if !request.DryRun {
 
-		// Add the cloned services.
-		for _, svc := range request.Cloned {
+		// Add the added services.
```
Comment (iancmcc) [inline facade/service.go]: This code matches the order from before, but I don't understand: why don't we want to apply our existing state BEFORE we add the new services? Or rather, why is it safe to do it in this order?

**Comment by wwwtyro** [comment]
Test this

**Comment by wwwtyro** [comment]
retest this

**Comment by wwwtyro** [comment]
retest this

**Pull request merged**